writefile_example.ipynb

```diff
@@ -3,7 +3,7 @@
   {
```

Comment:
<p>Why did you change to a slower cluster init method?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LauraRichter/examples/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>